src/ui/types/exporter-broker.d.ts

```diff
@@ -0,0 +1,12 @@
+interface ExporterBroker {
```

Comment:
Don't think we need this, we can just export in the above, `types/application`

Reply:
Using this for my mock body, without it, eslint goes mad

Reply:
It's the same thing though just without `id`, can we not just import from `types/application`? I think we're doing this in other places